backend/src/main/java/gov/cdc/usds/simplereport/api/model/User.java

```diff
@@ -1,59 +1,71 @@
 package gov.cdc.usds.simplereport.api.model;
 
 import java.util.Optional;
+import java.util.ArrayList;
 import java.util.List;
 
 import gov.cdc.usds.simplereport.config.authorization.UserPermission;
 import gov.cdc.usds.simplereport.db.model.ApiUser;
 import gov.cdc.usds.simplereport.db.model.Organization;
-import gov.cdc.usds.simplereport.db.model.auxiliary.PersonName;
+import gov.cdc.usds.simplereport.service.model.CurrentOrganizationRoles;
 
 public class User {
 
-	private String id;
+    private ApiUser wrapped;
 	private Optional<Organization> org;
-	private PersonName nameInfo;
-	private String email;
 	private Boolean isAdmin;
+    private String roleDescription;
 	private List<UserPermission> permissions;
 
-	public User(ApiUser user, Optional<Organization> org, Boolean isAdmin, List<UserPermission> permissions) {
+    public User(ApiUser user, Optional<CurrentOrganizationRoles> orgwrapper, boolean isAdmin) {
+        this.wrapped = user;
+        this.org = orgwrapper.map(CurrentOrganizationRoles::getOrganization);
+        this.permissions = new ArrayList<>();
+        this.isAdmin = isAdmin;
+        if (orgwrapper.isPresent()) {
+            permissions.addAll(orgwrapper.get().getGrantedPermissions());
+            roleDescription = orgwrapper.get().getEffectiveRole().get().getDescription();
+        }
```

Comment:
super nit-picky; feel free to ignore

```suggestion
        orgwrapper
            .map(CurrentOrganizationRoles::getGrantedPermissions)
            .ifPresent(permissions::addAll);
       this.roleDescription = orgwrapper
           .flatMap(CurrentOrganizaitonRoles::getEffectiveRole)
           .map(OrganizationRole::getDescription)
           .orElse(null);
```

Reply:
Nice! But I think not a today change, at 5:00 PM on the day it's due. 😁